--- chronojump-1.4.4.1/chronopic-tests/test-precision.c.orig	2013-04-12 13:20:03.000000000 +0200
+++ chronojump-1.4.4.1/chronopic-tests/test-precision.c	2014-06-22 17:01:23.143871975 +0200
@@ -158,7 +158,7 @@
            sprintf (cad,"Periodo: %7.2f ms, Freq: %2.3f Hz. ",ton+toff,
                     1000/(ton+toff));
            //sprintf (cad,"%X %X %X",trama[2],trama[3],trama[4]);
-           printf (cad);
+           fputs (cad, stdout);
            for (i=0; i<strlen(cad); i++)
              printf ("\b");
            fflush(stdout);
--- chronojump-1.4.4.1/chronopic-tests/test-saltos.c.orig	2013-04-12 13:20:03.000000000 +0200
+++ chronojump-1.4.4.1/chronopic-tests/test-saltos.c	2014-06-22 17:04:16.677201690 +0200
@@ -99,7 +99,6 @@
 /*----------------*/
 int main (int argc, char **argv)
 {
-  char cad[80];
   int serial_fd;
   int ok;
   double t;
@@ -187,8 +186,7 @@
           toff=t;
           
           //-- Imprimir informacion
-          sprintf (cad,"Tiempo: %8.1f ms\n",toff);
-          printf (cad);
+          printf ("Tiempo: %8.1f ms\n",toff);
         }
       case 1:  //-- Estado ON. Usuario estaba de la plataforma
         
